wandb/sdk/internal/internal_api.py

```diff
@@ -658,6 +658,8 @@ def fail_run_queue_item(
             "runQueueItemId": run_queue_item_id,
         }
         if "message" in self.fail_run_queue_item_fields_introspection():
+            if len(message) > 128:
+                message = message[:128]
```

Comment:
can't we always truncate safely?

```suggestion
            message = message[:128]  # max len
```

Reply:
same with below

Reply:
Ah yeah, I was initially planning a warning in the if, but then I decided against it